docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -132,7 +132,7 @@ public class Gift {
         this.name = name;
     }
 }
---
+----
 
 To load some SQL statements when Hibernate ORM starts, add an `import.sql` in the root of your resources directory.
```

Comment:
Suggestion - To load SQL statements when Hibernate ORM starts, add an `import.sql` file to the root of your resources directory. This script can contain any SQL DML statements. Make sure to terminate each statement with a semicolon.